taxi_service/urls.py

```diff
@@ -22,4 +23,9 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("", include("taxi.urls", namespace="taxi")),
-] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+    path("", include("taxi.urls", namespace="taxi")),
+    path("accounts/", include("django.contrib.auth.urls"))
+] + static(
+    settings.STATIC_URL,
+    document_root=settings.STATIC_ROOT
+) + debug_toolbar_urls()
```

Comment:
The `debug_toolbar_urls()` is included directly in the `urlpatterns`. Ensure that this is only enabled in a development environment, as it can expose sensitive information in production.